### PR TITLE
chore: Added script to fetch scene contents

### PR DESCRIPTION
**Pull request by pablitar**
Added script to fetch scene contents, for testing purposes

### CODE REVIEW

**Review by pbosio — APPROVED** [review]

File: kernel/scripts/fetchSceneContents.ts [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+}
+
+type Content = {
+  file: string
+  hash: string
+}
+
+async function main() {
+  const query = sceneId ? `id=${sceneId}` : `pointer=${pointer}`
+  const url = `${contentServerUrl}/entities/scene?${query}`
```
Comment (nchamo) [inline kernel/scripts/fetchSceneContents.ts]: We want to deprecate this API.

Let's try to use `/deployments`. You can use `entityType=scene` and `&entityId={sceneId}` or `&pointer={pointer}`

File: kernel/scripts/fetchSceneContents.ts [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+ *
+ * The scene will be downloaded from https://peer.decentraland.org/content. This can be overriden with the CONTENT_SERVER_URL variable
+ * The scene will be downloaded in the public/downloaded-scenes folder. This can be overriden with the OUTPUT_DIR variable
+ */
+
+import fetch from 'node-fetch'
+import * as fs from 'fs'
+
+const sceneId = process.env.SCENE_ID
+const pointer = process.env.POINTER
```
Comment (nchamo) [inline kernel/scripts/fetchSceneContents.ts]: Nit: Maybe calling it parcel would be clearer?

File: kernel/scripts/fetchSceneContents.ts (hunk 1 above)
Comment (nchamo) [reply kernel/scripts/fetchSceneContents.ts]: Actually, you could use the catalyst-client (I think it's already a dependency for kernel)

The API:
https://github.com/decentraland/catalyst-client

**Review by nchamo — APPROVED** [review]
Great work!!!

**Pull request merged**